examples/scmb/scmb.py

```diff
@@ -22,7 +22,7 @@
 # THE SOFTWARE.
 ###
 
-from hpOneView import *
+from hpOneView.oneview_client import OneViewClient
 from functools import partial
```

Comment:
Suggestion: Perhaps add a blank link to separate `from X import X` and `import X` types of imports